spinalcordtoolbox/scripts/isct_convert_binary_to_trilinear.py

```diff
@@ -113,28 +115,29 @@ def main():
 
     # upsample data
     printv('\nUpsample data...', verbose)
-    run_proc(["sct_resample",
-              "-i", "data.nii",
-              "-x", "linear",
-              "-vox", str(nx * interp_factor) + 'x' + str(ny * interp_factor) + 'x' + str(nz * interp_factor),
-              "-o", "data_up.nii"], verbose)
+    sct_resample.main([
+        "-i", "data.nii",
+        "-x", "linear",
+        "-vox", str(nx * interp_factor) + 'x' + str(ny * interp_factor) + 'x' + str(nz * interp_factor),
+        "-o", "data_up.nii",
+    ])
 
     # Smooth along centerline
     printv('\nSmooth along centerline...', verbose)
-    run_proc(["sct_smooth_spinalcord",
-              "-i", "data_up.nii",
-              "-s", "data_up.nii",
-              "-smooth", str(smoothing_sigma),
-              "-r", str(remove_temp_files),
-              "-v", str(verbose)], verbose)
+    sct_smooth_spinalcord.main(["-i", "data_up.nii",
+                                "-s", "data_up.nii",
+                                "-smooth", str(smoothing_sigma),
+                                "-r", str(remove_temp_files),
+                                "-v", str(verbose), ])
```

Comment:
I think the trailing comma here is unusual, since the closing brackets are on the same line. Using the same indentation style as a few lines below would be my favourite fix:
```suggestion
    sct_smooth_spinalcord.main([
        "-i", "data_up.nii",
        "-s", "data_up.nii",
        "-smooth", str(smoothing_sigma),
        "-r", str(remove_temp_files),
        "-v", str(verbose),
    ])
```

Reply:
Ah oops! That definitely wasn't intentional. :sweat_smile: